gradle/tests.gradle

```diff
@@ -70,16 +70,18 @@ task testng(type: Test) {
   systemProperties['selenide.browser'] = 'htmlunit'
 }
 
-tasks.withType(Test).all { testTask ->
-  testTask.systemProperties['file.encoding'] = 'UTF-8'
-  testTask.systemProperties['BUILD_URL'] = System.getenv()['BUILD_URL']
-  // testTask.maxParallelForks = 2
-  testTask.systemProperties += System.properties
-  testTask.jacoco {
+tasks.withType(Test) {
+  systemProperty('file.encoding', encoding)
+  systemProperty('BUILD_URL', System.getenv()['BUILD_URL'])
+//  maxParallelForks = Runtime.runtime.availableProcessors().intdiv(2) ?: 1
```

Comment:
commented code

Reply:
Was commented before changes. 
I'm only update script that calculate available processors.
Based on gradle performance guide: https://guides.gradle.org/performance/#running_tests